biocypher/write/graph/_neo4j.py

```diff
@@ -285,7 +285,11 @@ def _construct_import_call(self) -> str:
         import_call_neo4j_v5 = self._get_import_call(
             "database import full", "", "--overwrite-destination="
         )
-        import_script = f"# Neo4j v4 import call:\n{import_call_neo4j_v4}\n# Neo4j v5 import call:\n{import_call_neo4j_v5}\n"
+        neo4j_verison_check = (
+            f"version=$(neo4j-admin --version | cut -d '.' -f 1)"
```

Comment:
On my system I get as output for `version=$(neo4j-admin --version | cut -d '.' -f 1)`
`WARNING: Max 1024 open files allowed, minimum of 40000 recommended 4`. This will lead to failure in the next line. So it would be great to adapt the bash command to be more specific with the version selection.

Reply:
This may be related to your settings for the number of open files allowed? The recommended number by Neo4j is [40000 at least](https://neo4j.com/developer/kb/number-of-open-files-on-linux/
).

Reply:
Nice, thanks for the link. Now I have fixed my local config.